README.md

```diff
@@ -1,60 +1,87 @@
-# Which is the fastest?
+# Which is the fastest ?
```

Comment:
Some text and headers suggest only the speed is measured, but some other say that other metrics are also measured. It's not very consistent. I think it would be best to make it clear everywhere that this benchmark is not only about speed.